integrations/telegram/integration.definition.ts

```diff
@@ -20,7 +20,6 @@ export default new IntegrationDefinition({
       message: { tags: { id: {}, chatId: {} } },
       conversation: {
         tags: { id: {}, fromUserId: {}, fromUserName: {}, chatId: {} },
-        creation: { enabled: true, requiredTags: ['id'] },
```

Comment:
**issue:** don't remove these (same for `createUser`) in a patch, we'll wait and remove them all at once in a major bump. This is highly breaking

Reply:
sorry it wouldn't let me deploy locally when testing in my own workspace and forgot to add it back